azext_iot/central/models/ga_2022_05_31/template.py

```diff
@@ -0,0 +1,78 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+
+from azure.cli.core.azclierror import CLIInternalError
+
+from azext_iot.central.models import BaseTemplate
+
+
+class Template(BaseTemplate):
+    def __init__(self, template: dict):
+        super().__init__(template)
+        try:
+            self.id = template.get("@id")
+            self.interfaces = self._extract_interfaces(template)
+            self.schema_names = self._extract_schema_names(self.interfaces)
+            self.components = self._extract_components(template)
+            if self.components:
+                self.component_schema_names = self._extract_schema_names(
+                    self.components
+                )
+
+        except Exception:
```

Comment:
is there a more specific exception?

Reply:
No I used the same template model as old versions. They directly return this exception to users but not expose any other error messages like: schema parsing error, interface parsing error and etc. We can refactor this if more detailed message is necessary later.